lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -378,12 +375,9 @@ struct TypeLoweringVisitor : public FIRRTLVisitor<TypeLoweringVisitor, bool> {
   /// The builder is set and maintained in the main loop.
   ImplicitLocOpBuilder *builder;
 
-  // After a bundle type is lowered to its field elements, they can have a
-  // reference to a HierPathOp. This new lowered field element will get a
-  // new symbol, and the corresponding NLA needs to be updatd with this symbol.
-  // This is a list of the NLA name  which needs to be updated, to the new
-  // lowered field symbol name.
-  SmallVector<NlaNameNewSym> nlaNameToNewSymList;
```

Comment:
This is dead code after the changes here. All the modifications to `filterAnnotations` are due to the implications of removing this.